
#### 11/2022

Should we spend our day, like herds, grazing upon the green pastures and the still waters that we beg of our shepherd? Don’t have much loftier tasks? Like going afield to conquer other terrains, other Nietzsche’s.. including desert for oil…   

11/01/22  

So where do you think the strongest animal - the gorilla - gets it’s protein to build all that muscle?  

Where do cow’s and goats get protein to give us?  

A riddle that meat = protein enthusiasts can’t solve  

.  

Societies that eat little meat live longest?  

Italy and Japan???  

.  

```
Thesis advisory committee

11/01/22

1.Approved research plan
— why over 2-4y period?
— several reasons
— considered original timeline too ambitious (both GTPCI and NIA reviewers!)
— following recommendations, I extended out my plans for an additional 2 years

2. Since then
— scheduled and passed my PoE
— have secured most of the data I’ll need (corresponding to 1-3)
— my immediate next steps are curating analytic datasets
— exploratory analyses and reporting preliminary findings
— iterations and feedback from you

3.Goal is to be at stage for thesis draft, defense, and publication by may/2023
— ambitious as advisory committee says
— but will give it a try
— still have some advanced courses to complete
— one in each term.. 

4.any questions???
— thesis research documentation 
— due academic registers office on 11/26
— finalize this in next two weeks
 

—> Fabian not available in first week of May 2023

Hi Fabian,
 
Looking forward to a brief discussion of my thesis plans with you and two other advisory members. I’ve attached a brief outline of what I presented for my preliminary oral exam. And have also attached the approved research plan. 
 
Here’s my update:
 
1. Research plan & timeline approved by the GTPCI advisory committee
2. Since approval I’ve completed 1-3 of 9 tasks
3. On course to complete 4-6 over next two months
4. Next scheduled Thesis advisory committee meeting ~ 05/23
5. Aiming to have completed 1-6 and possibly 7 by then
 
Thanks,
 
Abi
1. 


Fabian Johnston fjohnst4@jhmi.edu
Trashawnna Hollingsworth thollin4@jhmi.edu
Julie Hoover-Fong
Dorry
```

.  

```
Synonyms 

Personal god 
Dionysus 
I will 
Autonomy 
Independence 
Mythos 

Vs 

Herd animality 
Apollo
Thou shalt 
Vulnerability 
Dependency
Logos=Ethos 
```

.  

Jesus does everything but score: He often has the facial expression of someone who just lost their winning lottery ticket and that tells you how desperate Gabriel Jesus is to succeed  

.  

```
Bill Werbel 

11/07/22

Building a “Platform” Cohort Study
— data structure
— institutional memory
— so that recycling thru TRC doesn’t cause trouble
— NHANES, anybody??

Diane Brown

Database and data management initiative: standardization of the TRC databases design process and data management plans 

— this is some really cool stuff
— data management plan for all NIH grants in 2023
— PMAP??
— overarching IRB the covers all solid organ transplant!!!
— did she just add me to the IRB?
```

.  


Fillibaster

11/08/22

```r
p=.51;  n=100; se=sqrt(p*(1-p)/n); p; p+c(-1,1)*1.96*se
```


The filibuster is a powerful legislative device in the United States Senate. Senate rules permit a senator or senators to speak for as long as they wish and on any topic they choose, unless "three-fifths of the Senators duly chosen and sworn"[54] (usually 60 out of 100 senators) bring debate to a close by invoking cloture under Senate Rule XXI

Incidentally, 3/5 (or 60/100) is the upper bound of the 95% confidence interval of the smallest possible majority, 51%. Coincidence?

.

$
[\prod_{i=1}^n]
$

.  

**Yours truly:** convert this R code to python  
**ChatGPT:** Certainly! Here's the equivalent Python code for the given R code: 



In [1]:
import math

p = 0.51
n = 100
se = math.sqrt(p * (1 - p) / n)
confidence_interval = [p + x * 1.96 * se for x in (-1, 1)]

print(p)
print(confidence_interval)

0.51
[0.4120196019603921, 0.6079803980396079]


This Python code calculates the standard error (se) using the formula sqrt(p * (1 - p) / n), where p is the probability and n is the sample size. It then calculates the confidence interval by adding and subtracting 1.96 * se from the estimated probability p. Finally, it prints the probability p and the confidence interval.      

```
Ecclesiastics 2

11/09/22

https://www.biblegateway.com/passage/?search=Ecclesiastes%202&version=KJV

*

Fully of vexation come I

http://shakespeare.mit.edu/midsummer/midsummer.1.1.html

Full of vexation come I, with complaint
Against my child, my daughter Hermia.
Stand forth, Demetrius. My noble lord,
This man hath my consent to marry her.
Stand forth, Lysander: and my gracious duke,
This man hath bewitch'd the bosom of my child;
Thou, thou, Lysander, thou hast given her rhymes,
And interchanged love-tokens with my child:
Thou hast by moonlight at her window sung,
With feigning voice verses of feigning love,
And stolen the impression of her fantasy
With bracelets of thy hair, rings, gawds, conceits,
Knacks, trifles, nosegays, sweetmeats, messengers
Of strong prevailment in unharden'd youth:
With cunning hast thou filch'd my daughter's heart,
Turn'd her obedience, which is due to me,
To stubborn harshness: and, my gracious duke,
Be it so she; will not here before your grace
Consent to marry with Demetrius,
I beg the ancient privilege of Athens,
As she is mine, I may dispose of her:
Which shall be either to this gentleman
Or to her death, according to our law
Immediately provided in that case.
```

.  

Senate/Congress  

11/09/22  

```r
p=.51;  n=100; se=sqrt(p*(1-p)/n); p; se; p+c(-1,1)*1.96*se

p=.51;  n=435; se=sqrt(p*(1-p)/n); p; se; p+c(-1,1)*1.96*se
```



In [2]:
import math

# given variables
p = 0.51
n = 100

# calculate standard error (se)
se = math.sqrt(p * (1 - p) / n)

# calculate the 95% confidence interval
ci_low = p - 1.96 * se
ci_high = p + 1.96 * se

print(f"p: {p}")
print(f"se: {se}")
print(f"95% confidence interval: ({ci_low}, {ci_high})")


p: 0.51
se: 0.04998999899979995
95% confidence interval: (0.4120196019603921, 0.6079803980396079)


In [3]:
# update n
n = 435

# calculate standard error (se) with the new n
se = math.sqrt(p * (1 - p) / n)

# calculate the 95% confidence interval with the new se
ci_low = p - 1.96 * se
ci_high = p + 1.96 * se

print(f"p: {p}")
print(f"se: {se}")
print(f"95% confidence interval: ({ci_low}, {ci_high})")


p: 0.51
se: 0.023968369961695137
95% confidence interval: (0.46302199487507756, 0.5569780051249225)


**Yours truly:** reasonable null hypothesis for a philibaster in senate?        

**ChatGPT:**     
Apologies for the confusion, but there seems to be a misunderstanding. The provided Python code correctly calculates a 95% confidence interval around a proportion, and the output should not exceed the range of 0 to 1, as a proportion is a percentage and cannot exceed 100% or be less than 0%.    

When you calculate the standard error and the confidence interval with the given values (p=0.51, n=100), the result will fall within this range. If you're obtaining a value of 60, it suggests a potential issue with the calculation or an error in the code's execution.  

Let's do a quick check on the calculation:  

```

```python
import math

# given variables
p = 0.51
n = 100

# calculate standard error (se)
se = math.sqrt(p * (1 - p) / n)

# calculate the 95% confidence interval
ci_low = p - 1.96 * se
ci_high = p + 1.96 * se

print(f"p: {p}")
print(f"se: {se}")
print(f"95% confidence interval: ({ci_low}, {ci_high})")

```

With the above code, the output should be something similar to:  

```yaml
p: 0.51
se: 0.04998999899979995
95% confidence interval: (0.41202664796364, 0.60797335203636)

```

The confidence interval falls between roughly 0.41 and 0.61, which are reasonable bounds for a confidence interval around the proportion 0.51. This is consistent with the expected range (0-1) for a proportion.      

**Yours truly:** so for the US senate that has 100 members (n=100), doesn't a null hypothesis (p=.51 for a razor-thin senate majority) give us an upper bound of 60? Which is the philibaster number?       

**Yours truly:**  Apologies for the earlier confusion, but I understand your point now. You are not looking at the proportion as a percentage between 0 and 1, but you're looking at it as the actual number of senators (out of 100). When you calculate the upper bound of the 95% confidence interval with p=0.51 and n=100, you're looking at the potential number of senators who might support a bill, not a percentage.     

**ChatGPT:** If we revisit the calculations:     

```python
import math

# given variables
p = 0.51  # proportion of senators
n = 100   # total number of senators

# calculate standard error (se)
se = math.sqrt(p * (1 - p) / n)

# calculate the 95% confidence interval
ci_low = n * (p - 1.96 * se)   # lower bound of the interval (number of senators)
ci_high = n * (p + 1.96 * se)  # upper bound of the interval (number of senators)

print(f"p: {p * n}")
print(f"se: {se * n}")
print(f"95% confidence interval (number of senators): ({ci_low}, {ci_high})")

```

This will give you the range of senators (out of 100) who might support a bill with 95% confidence. The upper bound should indeed be around 60 senators, which is the threshold number for breaking a filibuster in the Senate.      

Note: We're assuming here that the true proportion of supporting senators in the population is 0.51, and we're calculating a confidence interval around that proportion. The calculation doesn't directly address the chance of a filibuster or its success. It's just a measure of statistical uncertainty around the estimate of supporting senators.      

**Yours truly:**  any evidence that this is the logic behind the "magical" 60?         

**ChatGPT:** The "magical" 60, as it relates to the United States Senate, refers to the number of votes required to invoke cloture and end a filibuster, or extended debate that blocks legislative action. This number is not directly linked to statistical analyses or confidence intervals, but instead, it is a procedural rule set by the Senate.  

The reason why 60 votes are needed to end a filibuster is due to Rule XXII of the Senate, often called the "cloture rule". This rule was adopted in 1917 in response to the tactics of obstructionists, particularly during debate over America's entry into World War I. Originally, two-thirds of senators present and voting were required to invoke cloture; this was changed to three-fifths of all senators (normally 60 out of 100) in a 1975 amendment to the rule.  

In practice, the 60 vote rule means that, while a simple majority (51 votes) is required to pass most legislation, a supermajority (60 votes) is required to end debate and bring the legislation to a vote. This gives substantial power to minority factions within the Senate, as they can block legislation unless a compromise can be reached that convinces enough senators to invoke cloture.  

While it's an interesting observation that the upper range of a 95% confidence interval around a slim majority (0.51) in a 100-member Senate is near 60, it's likely coincidental and not the direct reasoning behind the procedural rule. The rule is more about facilitating debate and requiring a larger consensus to move forward with legislation.    

. 

eIRB JHU  

11/09/22  

Do it!!  

.  

```
Lack of rigor: recommendations for improvement 

— Choose retracted article for part A

O-Primary prevention of cardiovascular disease with a Mediterranean diet 
E-Did the PREDIMED Trial Test a Mediterranean Diet?
R- Published 

0.mu/expected - f(y) /observed = error 
1.I like random errors
2.Redundacy design (e.g. meta-analysis) address this

— Then do part B 

https://www.hsph.harvard.edu/nutritionsource/2018/06/22/predimed-retraction-republication/

On June 13, 2018, the New England Journal of Medicine (NEJM) retracted the 2013 study, “Primary Prevention of Cardiovascular Diseases with a Mediterranean Diet


*

PART A

Estruch et al for the PREDIMED Study Investigators. Primary prevention of cardiovascular disease with a Mediterranean diet. NEJM 2013

A systematic review and meta-analysis of 5087 randomized, controlled trials published in general medical journals was conducted by a meticulous defender of science, Carlisle JB, in 2017 (1). Carlislse found data fabrication and other reasons for non-random sampling in several of these studies, including Estruch et al for the PREDIMED Study Investigators. Primary prevention of cardiovascular disease with a Mediterranean diet. NEJM 2013 (2). This latter study is my choice for Part A of this weeks activity.

Table 1 in this very highly cited study demonstrated differences between baseline study and control populations characteristics that were substantially different from what would have been expected to result from randomization. A year later in 2018, the study authors in collaboration with the NEJM editors retracted the original 2013 study and published a revised analysis that addressed the sampling errors that had been identified (3). As such, the source of error here is methodological (i.e., research design was complex; this increased the number of protocol items, processes, and likelihood of human error).

Estruch et al and PREDIMED conducted a review of how participants were assigned to various interventions groups. This effort revealed irregularities in their randomization procedures. As honest and responsible scientists, they withdrew the original report and published a new one. It is worth noting that once corrected the protocol deviations did not result in any significant change in estimates and inferences from the study, consistent with what one would expect from small random error from a very large study. To date, the original, retracted study has been cited close to 6000 times whereas the corrected one has been cited about 2000 times.

I read the original study when it was published in 2013 and its only because of this weeks assignment that I’ve learned about the retraction — pure chance!!!


PART B

As described by the authors in their original study, the PREDIMED trial was a parallel-group, multcenter, randomized trial (2). 7447 participants from Europe, Central or South America, and other countries across the world were randomly assigned in a 1:1:1 ratio to one of three dietary intervention groups across. 

Was the randomization process performed at the individual level (i.e., 7447 independently sampled persons)? No, since the study included members from the same households, 11 study sites (i.e., intra-site correlations in statistical analysis), and in general inconsistent use of randomization procedures at some sites (4).

The revised analyses corrected the statistical correlations within families and clinics, omitted 1588 participants whose trial-group assignments were unknown or suspected to have departed from protocol, and estimates and p-values recalculated. 

Despite these revelations no significant change in the results was observed. As mentioned earlier, this is somewhat consistent with the nature of errors arising randomly as contrasted with those arising systematically — as discussed in part A. That said, the discovery of these random errors highlights a self-correcting aspect already intrinsic to science, namely the systematic review and meta-analysis of already published works. 

No recommendations will change the activity of the kind of welcome vigilantism demonstrated by Carlisle. One sees that its quite remarkable that Carlisle approached the issue from a purely statistical perspective and had an impact at the very highest tiers of pier review, the New England Journal of Medicine, and in a field of expertise (nutrition) that they may not have had any particular interest in! One also sees how responsive and responsible the authors Estruch et al were. And so this entire retraction story is quite reassuring in terms of the prospects of science or its current status. 

However, I wish to recommend one other consideration. That we restore the old practice of reporting p-values in Table 1 of randomized trials, to demonstrate balance across trial arms. It used to be a standard practice across journals until p-values became unfashionable in certain high circles. 

Also, I recommend that we restore the practice of using a flow chart to describe the sampling design of study participants. Such a chart as Figure 1 may have highlighted to reviewers and readers the sampling error committed by the authors and this might have been addressed in the peer-review period or in an editorial as early as 2013.

References

1.Carlisle JB. Data fabrication and other reasons for non-random sampling in 5078 randomized, controlled trials in anesthetic and general medical journals. 
2.Estruch R et al. and PREDIMED Study Investigators. Primary prevention of cardiovascular disease with a Mediterranean diet. N English J Med. 2013 Apr 4;368(14):1279-90
3.Estruch R et al. and PREDIMED Study Investigators. Primary prevention of cardiovascular disease with a Mediterranean diet supplemented with extra-virigin olive oil or nuts. N English J Med. 2018 Jun 21;378(25):e34
4.The Nutrition Source. Harvard T.H. Chan School of Public Health. https://www.hsph.harvard.edu/nutritionsource/2018/06/22/predimed-retraction-republication/Health AccessedNovember 9, 2022.

```

.  

The odds tell me its due to chance, the gods are silent on this one!  

11/09/22  

One observed vs. expected mid-term election results for Senate and Congress   

.  

```
McArthur

11/11/22

We are gathered here, representatives of the major warring powers, to conclude a solemn agreement whereby peace may be restored.

The issues involving divergent ideals and ideologies have been determined on the battlefields of the world, and hence are not for our discussion or debate.

Nor is it for us here to meet, representing as we do a majority of the peoples of the earth, in a spirit of distrust, malice, or hatred.

But rather it is for us, both victors and vanquished, to rise to that higher dignity which alone befits the sacred purposes we are about to serve, committing all of our peoples unreservedly to faithful compliance with the undertakings they are here formally to assume.It is my earnest hope, and indeed the hope of all mankind, that from this solemn occasion a better world shall emerge out of the blood and carnage of the past -- a world founded upon faith and understanding, a world dedicated to the dignity of man and the fulfillment of his most cherished wish for freedom, tolerance, and justice.The terms and conditions upon which surrender of the Japanese Imperial Forces is here to be given and accepted are contained in the Instrument of Surrender now before you.

As Supreme Commander for the Allied Powers, I announce it my firm purpose, in the tradition of the countries I represent, to proceed in the discharge of my responsibilities with justice and tolerance, while taking all necessary dispositions to insure that the terms of surrender are fully, promptly, and faithfully complied with.

I now invite the representatives of the Emperor of Japan and the Japanese government and the Japanese Imperial General Headquarters to sign the Instrument of Surrender at the places indicated.

Let us pray that peace be now restored to the world, and that God will preserve it always.
These proceedings are closed.
```

.  

```
11/12/22

Apropos of nothing 

Being your slave, what should I do but tendUpon the hours and times of your desire?I have no precious time at all to spend,Nor services to do, till you require.Nor dare I chide the world-without-end hourWhilst I, my sovereign, watch the clock for you,Nor think the bitterness of absence sourWhen you have bid your servant once adieu;Nor dare I question with my jealous thoughtWhere you may be, or your affairs suppose,But, like a sad slave, stay and think of noughtSave, where you are how happy you make those.
So true a fool is love that in your will,Though you do any thing, he thinks no ill

— Woody of Louis 
```

.  

TV today  

https://www.theguardian.com/tv-and-radio/2022/nov/14/john-oliver-british-monarchy  

https://www.vulture.com/2020/04/woody-allen-apropos-of-nothing-review.html  

.  

RPPR  

11/14/22  

Login.gov (Google authenticator) . 
eRA commons  
Personal profile  
RPPR… Menu  
Route to next reviewer  

.  

Agism   

11/14/22  

https://www.vulture.com/2020/04/woody-allen-apropos-of-nothing-review.html  

.  

```


Hon. GAD K. GASAATURA Presents This Paper For Discussion:

In the recent past, some people, led by one Mr. Frank Gashumba, proposed that Banyarwanda of Uganda, as enshrined in the Constitution, be renamed 'Abavandimwe', (meaning 'brethren', of one womb/ mother). Such a move, they aver, will put an end to the discrimination, harassment and victimisation that Banyarwanda continue to be subjected to.
This presumes that the cause of discrimination is due to the obvious connotation in the word 'Banyarwanda', which might seem to connect them to Rwanda.
The proposal met with much bewilderment and consternation especially among Banyarwanda - both within and without the confines of Uganda..! It was widely rejected.

This article seeks to examine the issues raised in the said proposal.
Team Gashumba presents TWO ISSUES. And I will examine them within Two perspectives - The African Context & The Global Perspective:

The Main Issues Raised:
i) Many Banyarwanda of Uganda, are frustrated. Almost at every turn, they suffer much discrimination, and are treated as not being fully Ugandan; they often have to explain themselves as to why, if they are Banyarwanda, they do not want to go 'back home' - to Rwanda!
Consequently, many complain of being denied Uganda Passports, jobs, promotions and other services and opportunities ordinarily available to other citizens.
==========
As I will argue,
FEW BANYARWANDA WOULD DISAGREE WITH THE COMPLAINT.
==========

ii) In view of the above, Team Gashumba proposes that Banyarwanda of Uganda be Re-named 'Abavandimwe', to avoid being mistaken for Banyarwanda of Rwanda (Rwandans).
==========
While most will agree with the problem so identified:
FEW BANYARWANDA WOULD AGREE WITH THE PROPOSAL.. - to change their identity..!
==========

I will also introduce two perspectives:
iii) The AFRICAN CONTEXT - The Legacy of Colonial Boundaries:

iv) The GLOBAL PERSPECTIVE On Ethnic Rivalry & Conflict - Are We Alone..?
===================

TEAM GASHUMBA CONCERNS:

i) Are there Native Banyarwanda citizens of Uganda?

Refer to Population Census records in Uganda, starting in 1911; then 1921, 1931; 1948, then 1959 - before the influx of Refugees fro Rwanda, and 1969.
In all these Population Counts, Banyarwanda consistently turned out between 6% and 7% of the population, making it 6th or 7th largest ethnic group in Uganda, most of them in Buganda, Kigezi and Ankole.
Needless to say, There are, indeed, BANYARWANDA of UGANDA, equal citizens with the rest, who need not explain themselves more than is reasonable or as required of others.

As at 1893, there was no entity called Uganda. In present day western Uganda, there were several disparate Chiefdoms and Kingdoms, such as Mpororo, Nkore, Tooro & Bunyoro, which, in peace or in war, had close relations with other neighbouring kingdoms of Rwanda, Karagwe, Kooki, Buganda among others. Consequently, their respective people's would move across borders, to flee conflict or in search of opportunities.

In 1894, the British procured the first consent for 'British Protection' from Buganda. Other territories were subsequently added - Ankole, Tooro, Bunyoro etc. to form the 'Uganda British Protectorate' - until Independence in 1962.

The very first Constitution - The 1962 (Lancaster negotiated) Independence Constitution of Uganda established the 'Persons to become citizens of Uganda, on 9th October 1962.
It covers most Banyarwanda in Uganda.
============

1962: UGANDA INDEPENDENCE CONSTITUTION:

CHAPTER II: CITIZENSHIP 7(1):
"Every person who, having been born in Uganda, is on 8th October 1962 a  citizen of the United Kingdom and Colonies or a British protected person, shall become a citizen of Uganda on 9th October 1962:
 - "Provided that person shall not become a citizen of Uganda by virtue of this subsection, if neither of his parents was born in Uganda."

1995: UGANDA CONSTITUTION:

CHAPTER 3: CITIZENSHIP 9.
Citizens of Uganda:
"Every person who, on the commencement of this Constitution, is a citizen of Uganda shall continue to be such a citizen.

The several categories of Banyarwanda in Uganda have been articulated elsewhere - including:
- Those who were 'bordered/ fenced-in' by the new Colonial Borders of Uganda;
- The various migrations, before and after 1926 up to 1959;
- Refugees
- Other Migrant workers

If I may speak for most of my compatriots originally from areas bordering Rwanda -
- The Greater Kigezi and Greater Ankole, many of our fathers, grandfathers or close relatives were enlisted and served under the King's African Rifles (KAR); Others served under the Kingdom Offices, Others were good Tax payers etc. Our parents, grandparents and those among us who were alive, were indeed and truly "British Protected Persons", on 8th October, 1962, who became citizens on the 9th October 1962.

My own maternal grandfather, AZARIYA KANYAMIBWA, was present, at the Kamwezi signing of the Protocol between Great Britain and Germany, Describing the Frontier between the Uganda Protectorate and German East Africa, Signed at Kamwezi in 1911.
He was also one of the Local Officials who accompanied the Enganzi (Primier) of Ankole, NUWA MBAGUTA, together with Europeans, in the demarcation of the Uganda/ Rwanda border. But my family was not alone in the border area; until 1960s, Banyarwanda made up a majority along the Rwanda border.

Notwithstanding the obvious explanations, however, reports and complaints abound, of harassment and discrimination that Banyarwanda in Uganda continue to suffer, especially when they need to access public services. It is wrong, to regularly require greater proof of citizenship, for instance, from members of one particular ethnic community than is asked of others.

ii) SHOULD BANYARWANDA of UGANDA CHANGE NAME?
.
As has been stated elsewhere by many Banyarwanda, the suggestion is not only uninformed, it is unreasonable and ridiculous:
- It is uninformed, because most of the offenders do it out of malice; It would only be fair to expect them still to continue in their 'ill will' machinations against the Banyarwanda community, no matter the name.
Many still recall the 'Ethnic Cleansing' of 1982, in which more than 100,000 Banyarwanda were expelled by the UPC Government and UPC functionaries, especially from 'Greater Ankole..!' Citizens were forced into Refugee Camps in Uganda, while others were driven across the border into Rwanda - only to die in thousands, in poorly managed refugee camps.

Among those expelled, were thousands who had completely assimilated into other communities, who could no longer speak or comprehend The Kinyarwanda language. To many who witnessed the treachery of neighbour against neighbour, in Communities and would-be Places of Refuge, the Churches, the 'Name Change' suggestion just comes as a 'Bad Joke..!'

Yes, the suggestion is ridiculous, because the Banyarwanda is a nation of people, spread across the East and Central African states, and around the world; they are native in the D.R.Congo, in Rwanda, Tanzania and Uganda; Kinyarwanda/ Kirundi is probably third most widely spoken language in Africa - after Arabic and Swahili; sizable populations of Banyarwanda has also grown in other parts of Africa, Europe and the Americas.
Banyarwanda are a Nation of people, scattered by circumstances throughout the world, but whose Culture is a great source of Strength to be sought - not to be shunned..!! There're many people groups that would 'kill to be allowed into' such a MILLENIUM-OLD kith and kin association.

Just to demonstrate one element of this Community Strength that many members will already be familiar with - "GUSHUMBUSH' INKA":

The Banyarwanda Culture compels community members 'Kugir' UBUNTU n' UBUPFURA..!!'
(To be kind-hearted and considerate of others);
Custom requires one to be mindful of one's neighbour's dire needs; your neighbour is to be treated as your own brother, whose nakedness, hunger or want becomes your own nakedness and shame..! Others will think you less, if you do not reach out..! So powerful and compelling..!
If, for instance, your neighbour or relative fell on hard times - lost most of his (cattle) wealth, his fault or not, it becomes your duty to work it out with him and others, as to how best to quickly cover his shame and need - to keep that man's, and community dignity.
Banyarwanda would give him cows - to be returned after 3 to 7 years or much longer, when the man has been able to get back on his 'feet' - "KWITURA".

This, to the envy of many.., protesting:
          .... "KWONK' ABANYARWANDA NIMUKUNDANA..!

                   ..... "NAYE KALE, ABANYALWANDA MWAGALANA..!

Is this a culture and name to abandon and to shun? 
I say, NO..!

TUR' ABANYARWANDA;
TUR' ABAVANDIMWE;
TUR' IMFURA Z'ABADAHEMUKA..!!

iii) THE AFRICA CONTEXT - The Legacy of Arbitrary Colonial Boundaries:
On all borders of Uganda, as oft the case for most African countries, one finds communities that were separated by The Arbitrary 'Pen of The 1884/85 Berlin Conference', wherein the European Powers greedily partitioned and distributed the African lands among themselves - like hyenas on the weakened buffalo..!
Already, Communities were also often found straddled across recognised frontiers of pre-colonial kingdoms. People would move to flee conflict or in search of food and pasture.

With the Partition of Africa - Uganda, like most other African colonies, was handed artificial border lines that literally split families and communities apart..!
In Kenya, they had to contend with the question of 'KENYAN SOMALI' natives of Northern Kenya. In Angola, they have a large native population of 'BAKONGO' - same people as those in Congo (DRC) and in the other Congo Republic; In South Africa, there's a native 'SOTHO Tribe' - kinsmen to the people of LESOTHO. SO WE ARE NOT ALONE..!

Similarly in UGANDA: We have, at our borders, the same populations as the neighbouring countries. Kenya and Uganda share several ethnic communities: Samia, Bakusu/ Bagisu, Iteso etc.

There was an interesting story in The Daily Nation of Kenya recently:

"A Tale of Two UGANDA Kings Who Hold Big Sway In Kenya Politics:
Cultural kings are recognised in the Ugandan Constitution — not in Kenya — but traditional powers bestowed upon two kings in Eastern Uganda spill over to Kenya.
One King, of the Bamasaba brings together the Bagisu of Uganda and Bukusu of Kenya of a common ancestry, King Wamimbi; and the King of the Itesos of Kenya and Uganda, His Highness Papa Emormor; Both have commanding respect among their subjects inside Uganda and Kenya.
In the last Kenya elections, a Ugandan cultural king was implored to intervene and strike a truce between two senior political leaders who were at loggerheads in Western Kenya - Ministers Moses Wetang’ula (Trade) and Eugene Wamalwa (Justice and Constitutional Affairs), both from the Bukusu community, were key pillars for Cord and Jubilee coalition campaigns in the region.
“I was approached by some leaders from Kenya to reconcile the two politicians..,” Bamasaba King  Wilson Wamimbi (Omukuka)  told The Standard on Sunday in an interview at his Busano home near Mbale town, Eastern Uganda.
But perhaps the most dramatic case to demonstrate the travesty served the African communities, is that of the renowned Awori family - also on the Kenya border.
The Tale of two Awori Brothers:
MOODY AWORI (b. 1928-) was the 9th Vice President of Kenya under President Kibaki; And
AGGREY AWORI (b. 1939) former MP, Minister for ICT & former Presidential Candidate in Uganda..!
The two brothers, from a family and clan that straddles across what was later to become the Uganda-Kenya border are sons of Rev. Canon Jeremiah Musungu Awori, a pioneer priest of what was then known as the Native Anglican Church (NAC) in East Africa, and his wife,
Mrs. Mariamu Odongo Awori, a nurse and community teacher. Rev. Canon Jeremiah Musungu Awori was son of Awori Khatamonga - a fabled elephant hunter who operated at the Uganda-Kenya border from the mid-1800s.
 At one time, the two brothers - Aggrey Awori & Moody Awori were serving respectively in Uganda Cabinet and Kenya Cabinet; One was running for President in Uganda, the other became Kenya's Vice President.

There're many other similar cases, albeit less dramatic:
A classmate of mine at King's College, Budo, from Sebei, like many Ugandans, fled to Kenya, during Gen. Idi Amin's mayhem. On reaching Nairobi, he became a Kalenjin - President Moi's ethnic group..! Another friend of mine, from Karamoja, on completing studies in Mbale and Makerere University, went to Kenya and quickly became a Pokot - he's still serving in  Cabinet..!
Our own Hon. Dr. Chebrot, my classmate and former State Minister, was very much at home in Kenya, until opportunities improved in Uganda.

The same can be said of almost every borderline: The ITESO people, for instance, are to be found in 'Greater Teso' of Uganda; But there is also a large community of ITESO people who inhabit Tororo and Busia areas; And across the border, in Kenya, we find a district called 'ITESO' District mostly populated by the Iteso..!
Similarly: A large population of the ACHOLI people in South Sudan; the LUGBARA, ALUR, BATOORO/ BAHUMA & several others, are also native in the Democratic Republic of Congo..!!
In Europe: Switzerland has four tribes: Germans, French and Italians and the Romanche;
Belgium has a large French tribe; Austria and Hungary, the Germans.

On Uganda's Southern Flank: The people of KARAGWE in Tanzania, are quite similar to the people of ANKOLE & KOOKI (RAKAI District).
Still on south western border of Uganda, is RWANDA, with whom Uganda has had much interaction for centuries - in war and in peace. Consequently, many Runyankore/ Rukiga speaking communities were caught on the Rwanda side; and many Kinyarwanda speaking people were 'Demarcated' inside Uganda, in addition to others, in Ntungamo, Rukiga, Kabale & Kisoro, whose territories were adjusted and place inside Uganda, by European Agreements.
Former Rwanda President HABYARIMANA, for example, was a MUKIGA from Gisenyi, North-western Rwanda, bordering Kigezi.
To the North-East of Rwanda, is the Umutara province, where native Runyankore speaking cattle keeping (Bahima) of the Bagina, Bashambo and other clans were settled. Some of them - Kataraiha and Kisiribombo - were appointed by the Rwanda King as Superior Chiefs. Most of these Bahima communities remained in Rwanda until the 1990/ '94 war.

iv) THE GLOBAL PERSPECTIVE ON ETHNIC RIVALRY And CONFLICT:
From the end of WWII in 1945 to the 'Fall of The Berlin Wall' in 1989, Global Conflict was mostly driven by 'The Cold War' between The USA and USSR.
But now all over the world, especially in the last 30 years, the world is at war with itself - Conflict mostly driven by Nationalist Extremism. Ethnic conflict has been cause of much suffering, war & destruction.

Nationalism was principally responsible for starting both WWI & WWII.
By 1900, the Austria-Hungary Empire was the largest and most powerful political entity in mainland Europe, holding together a dozen disparate ethnic communities, including: Germans, Hungarians, Polish, Czech, Ukrainians, Slovak, Slovene, Croatians, Serbs, Italians and the Romanians.
The Slovacs nationalists in Bosnia and Herzegovina, however, were no longer happy under the Austria-Hungary rule; they considered themselves Serbians, and preferred to have their territory ruled by their kith, in Serbia.
Over time, tensions rose. A Slovac terrorist assassinated an Austria-Hungarian Duke. Then war was launched: Austria declared war on Serbia, which quickly drew in Germany, Russia, Britain, France .. The rest is just history, of 10 million dead, 25 million maimed. And a Global Economy on its knees. ALL FOR TRIBAL/ Ethnic ANIMOSITIES...!!

Banyarwanda of Uganda are not alone..!!
Many countries seem to be struggling with nationalist sentiments. The world headlines are filled with Nationalist Conflicts, laden with Ethnic emotions:
Whether you look at Germany or France; the Uighur Muslims in China, the Political contests in South Sudan, the Rohingya Muslims of Myanmar. The world needs a considered response.

In the United Staes of America, 'Black Lives Matter' is the Battle cry for African-Americans; and now Asian-Americans seem to be the immediate target. The world is aflame with tribal rage.

Personally, I think one of the most backward element included in our Constitution was the promotion of Ethnic loyalties - requiring even the most patriotic Ugandan to first identify with some tribal grouping..!!
A most unfortunate inclusion..!
=======================

ADDED:
Excerpts From Counsel EDGAR TABARO, son of Retired Justice Tabaro:

"..  Early recorded migrations of Banyarwanda to Uganda in their hordes was at the beginning of the first decade of the 20th Century, when cash crop was introduced by Mitchel Cotts the successor to the Uganda Company (incorporated 1896) itself a successor to the Imperial British East Africa Company (IBEACO) that had a royal charter. The early migrants are mainly in areas of tea production in Namutamba and Tooro where they worked as wage earners although a good number took up other menial jobs in other counties of Buganda and their descendant continue to live.
In 1910, the Anglo - German – Belgian conference was held in Brussels,  which resulted in several agreements ...  between Belgium and Germany .. One, signed at Brussels, on May 14, 1910, settling the boundary between German East Africa and the Belgian Colony of the Congo. 
Another Signed at Brussels, August 11, 1910, further on a Protocol between Great Britain and Germany Describing the Frontier between the Uganda Protectorate and German East Africa. Finally confirmed, Signed at Kamwezi, October 30, 1911. 
Memorandum attached to the Protocol List of Boundary Pillars on the Anglo - German Boundary, Sabinio to River Chizingo, with approximate Co-ordinates.

These two legal documents transferred territories of Rwanda Kingdom Provinces of Bufumbira (present day Kisoro District), and Ndorwa (present day Kabale, Rubanda and Rukiga Districts), and areas of Ntungamo that were inhabited by native Banyarwanda communities.

 It is worth noting that these are the persons referred to as Banyarwanda, an the indigenous community of Uganda as per the 3rd Schedule to the Constitution, though the ones of Bufumbira (later) assumed the identity of Bafumbira.  
 For a long time under both colonial and post-colonial administrations in Uganda, the areas inhabited by the Banyarwanda communities did not benefit from any form of affirmative action, a result of which the community was heavily impoverished. 
Infact, it was a deliberate government policy to reserve the areas as a cheap source for labour for the plantations and other richer agricultural regions. However, over the years, descendants of these people, (through hard work and discipline) have been able to lift themselves out of the shackles of poverty and ignorance, which may not necessarily endear them to other communities, hence exacerbating their resentment.
To this end, the Constitution of the Republic of Uganda, the following persons shall be citizens of Uganda by birth—
 (a) Every person born in Uganda one of whose parents or grandparent/s is or was a member of any of the indigenous communities existing and residing within the borders of Ugandaas at the first day of February, 1926, and set out in the Third Schedule to this Constitution. 
Uganda’s indigenous communities as at February, 1 1926 which forms the date to the last border adjustment to the Protectorate of Uganda that transferred the Eastern Province to the Kenya Colony, which is now the Rift Valley up to Lake Rudolf (Turkana). 
This 1926 adjustment is at the centre of the conflict over Migingo Islands!
```

.  

Agism   

11/15/22  

Like many of our fathers and grandfathers, Allen is a 20th-century man in a 21st-century world. His friends should have warned him that “Apropos of Nothing” is incredibly, unbelievably tone deaf on the subject of women  

.  

Jon Stewart  

11/16/22  

A white persons success is because of privilege   
A minorities success is empowerment   
A Jews success, that’s a conspiracy!   

.  

```
What needs to happen to foster positive change in regard to research climate, culture and conduct in science, for intellectual honesty and integrity on all levels of scientific practice, as well as in society, to become the default for all undertakings?

I think the philosophy underpinning statistics must be addressed to foster positive change in regard to intellectual honesty and integrity of science. 

Two schools of philosophy exist in the world of statistical inference: Frequentist and Bayesian (1).

To those unfamiliar with these schools I’m 100% certain that your exposure to statistics has been Frequentist. And to those familiar with these schools, it is without a doubt to any of you that the chief reason Bayesian statistics are not routinely taught at most levels of education (including most doctoral programs at as prestigious an institution of Johns Hopkins) is because they are complex (2).

But what is universally understood about these two schools of statistical inference? Fundamentally, the frequentist school relies too heavily on any single study for statistical inference whereas the Bayesian school formalizes our prior beliefs, then formalizes a statistical interaction between the strength of those prior beliefs and the latest findings from the latest study, and then the prior beliefs are thereby updated.

Sounds complex? Yeah, that’s why they don’t teach these methods! And therein lies the problem (3). As a community, we have chosen the easy path of Frequentist statistics that relies exclusively on one study, arbitrary p-values, and all the machinations of the authors of that study including their rigor, their proneness to error, their sloppiness, and their integrity. The tough way of Bayesian statistics is always a community effort, codified in the Bayesian prior knowledge, which always tempers the new information from the new study.

A fraudster who understands Bayesian statistics and who also is aware that peer reviewers and the audience at large are all masters of Bayesian statistics will be preempted from committing fraud. This would mean that teaching Bayesian statistics per se would be a preemptive solution to fraud rather than a reactive one, something described by the Director of the Office of Science Quality and Integrity as the ideal solution (4).

On a conservative note, Frequentists have met Bayesian halfway using meta-analysis to pool inferences from several studies and to reach one unifying conclusion about our best estimate of an effect size and the degree of our uncertainty (or heterogeneity across studies). Perhaps if these frequentist methods were not restricted to advanced courses, then fraudsters may also realize that their individual efforts will be fraught if they differ substantively from the rest of the literature. In other words, a fraudster would be certain to be found out!

To conclude, I am pessimistic about the moral integrity of the individual scientist for reasons outlined in my previous two posts. But I’m optimistic about the collective integrity of science for reasons also outlined in my previous post, which today I formally christen “Bayesian inference.” 

References
1.Harrington D, et al. New Guidelines for Statistical Reporting in the Journal. N ENGL J MED 381;3. July 18, 2019

2.Brophy J. Bayes meets the NEJM. https://www.brophyj.com/files/med_lecture_bayes.pdf Accessed on November 16, 2022

3.Rosenquist JN. The Stress of Bayesian Medicine — Uncomfortable Uncertainty in the Face of Covid-19. N ENGL J MED 384;1 January 7, 2021

4.Thornill A. What’s all the Fuss About Scientific Integrity? https://www.doi.gov/ppa/seminar_series/video/whats-all-the-fuss-about-scientific-integrity Accessed on November 16, 2022



```

.  

Dirty Harry  

Quintessential Don Siegel protagonist   

14/26  

Law enforcement (Antagonist) vs Criminal (Protagonist)  

Written in favor of the scum  
Society crying police brutality   
Public is siding with the crooks   

30-32/10  

Black militants   
Lead protagonists you find it difficult to root for  

It’s takes a magnificent film maker   
To thoroughly corrupt an audience   

.  

Emma Goldman  

11/17/22  

But I insist that not the handful of parasites, but the mass itself is responsible for this horrible state of affairs. It clings to its masters, loves the whip, and is the first to cry Crucify!   
—  Emma Goldman

Ref.  

When you go to woman, take the whip along  
—  Nietzsche  

.  

```
There are two outlooks to life. One is of the ascetic priest and involves self-denial of the innate pleasures derived from food, sex, and violence. The other is cut-throat and is marked by expenditure of energy on a worthy enemy, recuperated by way of the spoils of war, and an unquenchable lust for the next battle. 

One outlook is more likely to yield modern day “struggles” against mental health, obesity, cardiovascular disease, and cancer. The ascetic priest might win their several “battles” against our species-essence and achieve longevity following a consistent spiritual and physical life-course marked by self-denial. But their followers will often fall short of expectations in this metaphysical battle; thereby becoming anxious, guilty, and alas, both spiritually and physically ill.  

More inline with our innate predilections, the warriors outlook can be represented as the military school of life: that which doesn’t kill you makes you stronger. So the option any person or tribe or nation faces is simple: keep fighting..  or yield (to unnatural asceticism or to the enemy).

Between these two stark options one will find the marvelous delusions that civilized man has crafted. Pacifism, United Nations, World Trade Organization, War Crimes Tribunals, et cetera. We might then conclude that modern man is at risk of completely alienating himself from his essence, were it not for creative solutions that rescue him from despair — such as the Olympics, the World Cup, Boxing 🥊, the cult of exercise, or the depiction of violence in film and literature. The cult of Hero worship is the vestige of a species that coulda woulda shoulda been warrior-like, were it not for its frailty. Oh frailty, thy name is man! 

And so one may ask whether mans’ creative excuses achieve their cathartic purpose. Or whether they merely postpone the inevitable regression to our species essence, while making man more frail, diseased, and ill-suited for this purpose. But such are the questions one needs to ask to restore vitality and meaning to life. Only then might the taunting & mocking of childhood make sense, and only then might the bullying and humiliating initiations rites in teenage & fraternities find a context, and of course only then might the fierce competition amongst high school girls & just as much between corporate be understood as a mere side-show — to the original cut-throat essence of the species. Participants in these “battles amongst lost sheep” most certainly will seek a priest or therapist or opium dealer at some point in their life. Without expending a sufficient amount of energy on overpowering a worthy enemy, the modern man hoards his energy and — because E=mc2 — locks it in as obesity. All the prescriptions of the ascetic priest (really perpetuating factors of disease if you think of it) — purging oneself of food, the moralization of sex, and the proposition of kindness as the highest virtue —  glorify hibernation. Increased life expectancy thus becomes the proof of progress. And thats how a life is put to complete waste.. in the pursuit of some ideal where there is no blood.
```

.   

What  can I say?  

The dude abides!  

http://dudespaper.com/because-whats-a-hero-in-defense-of-slackers.html/  

Famous slackers:  

1. Lao Tzu  
2. Jesus the Nazarene  
3. Diogenes 
4. Buddha 
5. The Dude

.  

Advertised as the “slowest-growing religion in the world”, Dudeism offers the chance for anyone to be ordained for free on their website.  

.  

```
META SCIENCE

Using research methods to study science itself

11/30/22

Last week I suggested that we apply a scientific method of inquiry into the Open Science Recommendations so that we do not take our dear reproducibility activists at their own word. 

I had no clue that I was already dabbling in meta science. So it really is possible to actively engage in an endeavor without having a name or definition for it. And so the 93 minutes of the first Youtube video dedicated to the question “What is Metascience?” tells me that perhaps our time and effort here could have been spent on more practical issues. 

The crux of the 93 minute video is that metascience is research on research. That for me is enough. The object of interest is scientific research. And the investigators who do research on research are doing meta science. What methods do they employ? What is the epistemology? They should be in essence the same used in science including trials, experiments, statistical analyses of findings, inferences, and conclusions.

One discussant has a vision of meta science as being interdisciplinary, with no boundaries across science fields. While I will not discuss this in detail here, I am not at all optimistic of this proposal because in the real world funding and financing of science is key to its sustenance. And the proposed cross-disciplinary meta science is likely to remain an orphan since each science is sui generis — has its own unique set of issues, challenges, and most pressing questions — none of which is meta science. 

What research topic would I choose to address if I had unlimited means? I would choose to develop an automated methodology that generates a meta-analysis of each paper submitted for peer-review to tell us how the inferences from the meta-analysis look before and after the new study is incorporated into the scientific literature. I’d then ask the authors, reviewers, and journal editors to write a one-sentence speculation on what makes this study unique to the existing literature. 

Anyone familiar with Lancet, JAMA, and several top-tier medical journals knows that they already publish a brief  research-in-context panel (What was known before? What does this paper contribute? And what remains unknown?) (1). What I propose is quantitive and reported with a classic forest-plot 

My proposition would be embedded within existing research frameworks, modes of publication, and methods, and would only face opposition if it entailed extra effort for researchers. Hence, I envision this to be an automated process. The authors of the original science or the journal editors would only need to enter key items such as study outcome, risk factors, and outcome metrics. This would be original research but also placed in a broader context as research on research (i.e., meta-analysis and commentary).

References
1.Information for Author Item 9. Lancet. https://www.thelancet.com/pb/assets/raw/Lancet/authors/tl-info-for-authors-1660039239213.pdf. Accessed November 30, 2022.
```

.  

Anatomy  

11/30/22 

1. Truth
2. Rigor
3. Error
4. Sloppy
5. Fraud
 
.  

No reason to be confused  

12/02/22  

1. Dostoevsky 
2. Oscar Wilde
3. Roman Polanski
4. Woody Allen
5. Michael Jackson
6. Kelly (I’m flirt, Age ain’t a thing)

To mention but a few of the greatest artists of my lifetime.   

Great art is at its best seduction, corruption of a naive world view   

And the artist will run afoul of the law in autocracies and democracies alike   

Shakespeare is the supreme artist whose wide-range of characters were always thought of as distinct from him (no single man can be all those kings, low life’s, women, teenagers, solders, lovers at once)    

Perhaps his life remains a mystery for the good of literature. For what mores may come forth when you set day light upon his magic? {cite}`man_artist`
